### DISCUSSION TITLE
Error in "mutate"

### DISCUSSION THREAD

**Discussion by julianasr36** [Flavjack/GerminaR#10] [Q&A] [answered]
Hi! I've been trying to use the package to analyze my germination data without success. I am following the script provided, but It keeps returning the following error:

Error in `mutate()`:
! Problem while computing `grs = ger_GRS(evalName, data)`.
Caused by error in `FUN()`:
! invalid 'type' (character) of argument

Using the str command to check the data type and it is a mix of factors and num. Do you have an idea about what the problem is?

Thanks!

**Comment by Flavjack** [comment, marked as answer]
@julianasr36, thanks for writing,
If you follow the recommendations for the data collection: <https://germinar.inkaverse.com/articles/introduction.html> and check the code example <https://germinar.inkaverse.com/articles/GerminaR.html> and the problems persist. Could you share your dataset to check the possible problem? Please write to me at my email <flozano@lamolina.edu.pe>.

**Comment by julianasr36** [comment]
Thanks!
